manifests/server/puppetserver.pp

```diff
@@ -0,0 +1,29 @@
+class puppet::server::puppetserver (
+  $java_bin          = $::puppet::server_jvm_java_bin,
+  $config            = $::puppet::server_jvm_config,
+  $jvm_min_heap_size = $::puppet::server_jvm_min_heap_size,
+  $jvm_max_heap_size = $::puppet::server_jvm_max_heap_size,
+  $jvm_extra_args    = $::puppet::server_jvm_extra_args,
+) {
+
+  $puppetserver_package = pick($::puppet::server_package, 'puppetserver')
+
+  Augeas {
+    lens    => 'Shellvars.lns',
+    incl    => $config,
+    context => "/files${config}",
+    notify  => Service['puppetserver'],
+    require => Package[$puppetserver_package],
```

Comment:
Yeah, but the other way around as well, e.g. https://github.com/theforeman/puppet-puppet/blob/master/manifests/server/config.pp#L153-L154